There's a problematic chunk in our <time.h>:

#ifndef tzname
#define tzname _tzname
#endif

--- origsrc/pykde4-4.7.0/sip/nepomuk/ncal.sip	2011-05-20 15:33:41.000000000 -0500
+++ src/pykde4-4.7.0/sip/nepomuk/ncal.sip	2011-08-07 06:00:58.182150900 -0500
@@ -18,6 +18,7 @@
 // 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 %ModuleHeaderCode
+#define tzname tzname
 #include <nepomuk/ncal.h>
 %End
 
